### DISCUSSION TITLE
Custom roles [Jul 21]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2868] [Discord Threads]
> 🙋‍♂️ **rohitstartsup**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@903377031255965747> thanks for the shout-out! currently, NocoDB doesn't support custom roles. standard supported roles are listed here    
####    
#### https://docs.nocodb.com/setup-and-usages/team-and-auth#user-role-permissions  
#  
> 👨‍🦲 **dstala**  
#### To be specific, I am looking to disable certain columns from being updated by editor-roles. Is this possible?   
####    
#### > not supported at the moment.  
#  
> 🙋‍♂️ **rohitstartsup**  
#### Thanks for the heads up!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/999300005280104508)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
